webservices/legal_docs/responses.py

```diff
@@ -0,0 +1,709 @@
+                            }
+                        ],
+                        "doc_id": "mur_7132",
+                        "url": "/legal/matter-under-review/7132/",
+                        "respondents": ["Pitts, Michael David", "Evergreen Incentives LLC"],
+                        "name": "MICHAEL DAVID PITTS",
+                        "election_cycles": [2014],
+                        "commission_votes": [
+                            {
+                                "vote_date": "2018-03-01T00:00:00",
+                                "action": """
+The Commission decided by a vote of 4-0 to take the following actions in MUR 7132:
+1.  Accept the conciliation agreement with Michael David Pitts, as recommended in the Memorandum to the Commission
+dated February 14, 2018.
+2.  Approve the appropriate letter.
+3.  Close the file.
+"""
+                            },
+                            {
+                                "vote_date": "2017-05-10T00:00:00",
+                                "action": """
+The Commission decided by a vote of 4-1 to take the following actions in \nPre-MUR 596 and MUR 7132:
+1. Transfer Pre-MUR 596 to the Office of Alternative Dispute Resolution.
+2. Find reason to believe that Michael David Pitts knowingly and willfully violated 52 U.S.C. \u00a7 30102(b)(3) and
+11 C.F.R. \u00a7 102.15.
+3. Enter into pre-probable cause conciliation with Michael David Pitts prior to a finding of probable cause to believe.
+4. Approve the Conciliation Agreement for Michael David Pitts, as recommended in the First General Counsel\u2019s Report
+dated March 6, 2017.
+5. Approve the Factual and Legal Analysis for Michael David Pitts, as recommended in the First General Counsel\u2019s
+Report dated March 6, 2017.
+6. Approve the appropriate letter.
+"""
+                            }
+                        ],
+                        "mur_type": "current",
+                        "participants": [
+                            {"role": "Primary Respondent", "citations": {}, "name": "Pitts, Michael David"},
+                            {"role": "Complainant", "citations": {}, "name": "AMEDISYS, INC."}
+                        ],
+                        "highlights": [
+                            " to carry out his  <em>embezzlement</em> scheme.  ",
+                            "  is prohibited by law.  6. Pitts sought  to conceal the  <em>embezzlement</em> by under",
+                            "  execute  his various  <em>embezzlements</em>. That  month, Mr.  Pitts also  transferred",
+                            " that  was used to carry out his  <em>embezzlement</em> scheme.  Pitts also  did not  ",
+                            " the  <em>embezzlement</em> by under-reporting the  transfer of payroll  deductions from"
+                        ],
+                        "document_highlights": {}
+                    }
+                ],
+                "total_murs": 2,
+                "admin_fines": [],
```

Comment:
Can we take out these empty `admin_fines` and `adrs` lists, now that the examples have been fleshed out above?

Reply:
Did you mean take them out, or provide examples of them _if_ the query did return ADRs and Admin fines?